test/unit/org/apache/cassandra/service/StartupChecksTest.java

```diff
@@ -103,6 +103,13 @@ public void compatibilityCheckIgnoresNonDbFiles() throws Exception
         startupChecks.verify();
     }
 
+    @Test
+    public void checkReadAheadKbSettingCheck() throws Exception
+    {
+        startupChecks = startupChecks.withTest(StartupChecks.checkReadAheadKbSetting);
+        startupChecks.verify();
```

Comment:
Not sure if there should be some sort of assert, because this doesnt test much.

Reply:
Yeah this is ok, it's just testing the check will not throw any exception. Maybe add a little comment about this?

Reply:
Added.